src/components/ModelTableList/shared.tsx

```diff
@@ -96,16 +125,24 @@ export function getStatusValue(status, key) {
                 className="u-flex--block p-tooltip--top-center"
                 aria-describedby="tp-cntr"
               >
-                <div className="has-icon">
-                  <i className="p-icon--machines"></i>
-                  <span>{machineCount}</span>
-                </div>
+                {extra
+                  ? generateModelDetailsLink(
+                      status.model.name,
+                      extra,
+                      <div className="has-icon">
+                        <i className="p-icon--machines"></i>
+                        <span>{machineCount}</span>
+                      </div>,
+                      "machines",
+                      "p-link--soft"
+                    )
+                  : null}
                 <span
```

Comment:
Could you replace these tooltips with react-components.

Reply:
Done!